internal/translator/testdata/basic_include/output.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: projectcontour.io/v1
+kind: HTTPProxy
+metadata:
+  creationTimestamp: null
+  name: delegation-root
+  namespace: default
+spec:
+  includes:
+  - conditions:
+    - prefix: /service2
+    name: service2
+  routes:
+  - conditions:
+    - prefix: /
+    services:
+    - name: s1
+      port: 80
+  virtualhost:
+    fqdn: root.bar.com
+status:
+  currentStatus: ""
+  description: ""
```

Comment:
If there isn't an issue to set up yaml lint can you please add one; it will squeal about these missing \n's. It must be a vscode thing.

Reply:
You already added #10 - I'll do that next.